packages/lib/modules/pool/actions/create/PoolCreationFormProvider.tsx

```diff
@@ -52,6 +66,14 @@ export function usePoolFormLogic() {
     poolCreationForm.setValue('poolTokens', updates)
   }
 
+  const invertReClammPriceParams = () => {
+    const { initialMinPrice, initialMaxPrice, initialTargetPrice } = reClammConfigForm.watch()
+    reClammConfigForm.setValue('initialMinPrice', invertNumber(initialMaxPrice))
+    reClammConfigForm.setValue('initialMaxPrice', invertNumber(initialMinPrice))
+    reClammConfigForm.setValue('initialTargetPrice', invertNumber(initialTargetPrice))
+    updatePoolTokens([...poolTokens].reverse())
+  }
```

Comment:
have to invert reclamm params in LS state for all the config to display properly and allow user to update inverted values